tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -108,31 +108,21 @@ def _create_find_statistics_snapshot(self) -> str:
                 lambda driver: driver.find_element(By.XPATH, "//*[@data-testid='publicationsList']")
             )
 
-            publications_list = driver.find_element(By.XPATH, "//*[@data-testid='publicationsList']")
-
-            all_publications = publications_list.find_elements(By.CSS_SELECTOR, "li")
-
-            for publication in all_publications:
-                publication.click()
-
-                publication_details = publication.find_element(By.CSS_SELECTOR, "dl")
+            publications_list_items = driver.find_elements(By.XPATH, "//*[@data-testid='publicationsList']/li")
```

Comment:
So I guess previously the old xpath selectors using the double slash prefixes were essentially breaking out of their parent elements and searching from the root of the DOM - and returning the first match they find.

Reply:
Yes that's correct.

I believe that although `//` is a descendent selector, when it was used at the beginning of the statement
`publication_details.find_element(By.XPATH, "//*[dl]//div//dd[@data-testid='theme']")` the `//` doesn't mean descendent of the `publication_details` element, it's relative from anywhere on the page.

By changing expressions to begin with `./` we now select relative from the current element.